junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvSource.java

```diff
@@ -50,10 +50,21 @@
 
 	/**
 	 * The column delimiter to use when reading the {@linkplain #value lines}.
+	 * This can't be set simultaneously with {@link #delimiterString delimiterString}.
+	 * If neither attributes are set, the delimiter will be ','.
 	 *
-	 * <p>Defaults to {@code ','}.
+	 * <p>Defaults to {@code '\0'}.
 	 */
-	char delimiter() default ',';
+	char delimiter() default '\0';
+
+	/**
+	 * The column delimiter to use when reading the {@linkplain #value lines}.
+	 * This can't be set simultaneously with {@link #delimiter delimiter}.
+	 * If neither attributes are set, the delimiter will be ','.
+	 *
+	 * <p>Defaults to {@code ""}.
```

Comment:
I think we should remove the defaults from the Javadoc because they might confuse users.

Reply:
Indeed, I was thinking the same thing